targets/tools/js2c.py

```diff
@@ -77,10 +81,16 @@ def removeWhitespaces(code):
 OUT_PATH = './source/'
 SRC_PATH = './js/'
 
+parser = argparse.ArgumentParser(description="js2c")
+parser.add_argument('build_type', help='build type', default='release', nargs='?')
+parser.add_argument('--ignore', help='files to ignore', dest='ignore_files', default=[], action='append')
+parser.add_argument('--no-main', help='don\'t require a main.js file', dest='main', action='store_false', default=True)
```

Comment:
You can eliminate the need for escaping the `'` character if the string starts with the `"` character (and also ends with it): `"don't require..."`